### ISSUE TITLE
Bug: Deleting Indices Older Than 90 Days with Curator

### ISSUE THREAD

**Issue by MarcinNaw** [dtag-dev-sec/tpotce#116]
In /etc/crontab we can find:

```
# Delete elastic indices older than 90 days (kibana index is omitted by default)
27 4 * * *      root    docker exec elk bash -c '/usr/local/bin/curator --host 127.0.0.1 delete indices --older-than 90 --time-unit days --timestring \%Y.\%m.\%d'
```

However, running this command manually yields an error:

```
sudo docker exec elk bash -c '/usr/local/bin/curator --host 127.0.0.1 delete indices --older-than 90 --time-unit days --timestring \%Y.\%m.\%d'
Error: no such option: --host
```

I think curator uses now yaml configuration files, we should switch to them.

**Comment by t3chn0m4g3** [comment]
Thanks, it is a duplicate of issue #22 though. I noticed it a few weeks back and I am still undecided if we should stick with curator (and presumably being surprised by another sudden change for usage) or work out something else. I have been testing with some Elastic functions as well as simple deletes.
17.06 will be released in a few stages, I am still waiting on ASN databases for ELK 5.x (it's just around the geoIP plugin corner :) to work before releasing anything else but an alpha. Which means lots of opportunities for improvements.

**Comment by t3chn0m4g3** [comment]
Fixed in 8e8f94b1b4a67e11736538be8811b7e1cea1599a for T-Pot 17.x
